src/Chapter06.Tests/Listing06.35B.Tests.cs

```diff
@@ -0,0 +1,17 @@
+﻿using AddisonWesley.Michaelis.EssentialCSharp.Shared.Tests;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter06.Listing06_35B.Tests;
+
+[TestClass]
+public class ProgramTests
+{
+    [TestMethod]
+    [Ignore]
```

Comment:
Does this still need to be ignored?

Reply:
These are still not passing and I didn't investigate.